### PR TITLE
refactor(framework): EO-1141 Refactor Tables Sortable Column

### PR DESCRIPTION
**Pull request by satyamkotakonda**
JIRA: https://jira.rax.io/browse/EO-1141

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- ~~[ ] QE LGTM (no change to functional javascript)~~

Before
---------

After
-------

### CODE REVIEW

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <table rx-floating-header>
+    <thead>
+      <tr>
+        <th scope="col">
+          <rx-sortable-column
+            sort-method="sortCol(property)"
+            sort-property="name"
+            predicate="sort.predicate"
+            reverse="sort.reverse">
+              Name
```
Comment (khaziwallis) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: @satyamkotakonda i think this two space is not required.

Comment (satyamkotakonda) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: @CITguy  Please confirm.

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            predicate="sort.predicate"
+            reverse="sort.reverse">
+              Testing Sort Errors (see Protractor Tab)
+          </rx-sortable-column>
+        </th>
+      </tr>
+    </thead>
+    <tbody id="talentPoolData">
+      <tr ng-repeat="resource in talentPool | rxSortEmptyTop:sort.predicate:sort.reverse">
+        <th scope="row" class="talent-name">
```
Comment (KalpeshBhandari) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: Each attribute on separate line. In this case class can go in next line.

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<div ng-controller="rxSortableColumnSimpleCtrl">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: We can simplify by moving the `ng-controller` to the `<table>` on line 2.

1. Please move `ng-controller` to `<table>` tag on line 2.
2. Please remove `<div>`

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html (hunk 1 above)
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: "Name" should be at same indentation as attributes above.

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html (hunk 2 above)
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: This case is fine as is. It falls well within line length constraints and is easy to read.

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  <table rx-floating-header>
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: Please add `class="sortable-column-table"` for use with midway CSS selector.

File: utils/rx-page-objects/test/rxSortableColumn.midway.js [hunk 5]
```diff
@@ -11,7 +11,7 @@ describe('rxSortableColumn', function () {
     var columns, nameColumn, roleColumn;
 
     before(function () {
-        demoPage.go('#/components/rxSortableColumn');
+        demoPage.go('#/elements/Tables');
         columns = rxSortableColumn.byTable($('.module-demo table'));
```
Comment (CITguy) [inline utils/rx-page-objects/test/rxSortableColumn.midway.js]: Update line to:

```javascript
columns = rxSortableColumn.byTable($('.sortable-column-table'));
```

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+            sort-method="sortCol"
+            sort-property="none"
+            predicate="sort.predicate"
+            reverse="sort.reverse">
+              Testing Sort Errors (see Protractor Tab)
+          </rx-sortable-column>
+        </th>
+      </tr>
+    </thead>
+    <tbody id="talentPoolData">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: Please remove `id` attribute.

**Comment by ssalinas24** [comment]
Design LGTM

**Comment by satyamkotakonda** [comment]
@CITguy  Unable commit the changes and there is a lint issue in rxDatePicker. exercise. j's
Can you please help me here.

File: src/elements/Tables/docs/examples/rxSortableColumn.simple.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
+<table rx-floating-header ng-controller="rxSortableColumnSimpleCtrl" class="sortable-column-table">
```
Comment (CITguy) [inline src/elements/Tables/docs/examples/rxSortableColumn.simple.html]: Please separate attributes on their own line.

```html
<table
  rx-floating-header
  ng-controller="rxSortableColumnSimpleCtrl"
  class="sortable-column-table">
```

**Comment by satyamkotakonda** [comment]
@CITguy  Updated changes and please review.

**Comment by bry4n** [comment]
PR looks good to me. Dev LGTM

**Pull request merged**